<launch>
	<!-- node name="configure" pkg="flircam" type="configure.py" / -->
	<node name="SerialPortROSBridge" pkg="flircam" type="SerialPortROSBridge.py" respawn="true" />
	<node name="pantilt" pkg="flircam" type="pantilt.py" respawn="true" />
	<!-- node name="telemetry2" pkg="flircam" type="telemetry2.py" respawn="true" / -->
	<!-- node name="alert" pkg="flircam" type="alert.py" respawn="true" / -->
	<node name="ImageProc" pkg="flircam" type="ImageProc" respawn="true" />
	<node name="FrameGrabber2" pkg="flircam" type="FrameGrabber2" respawn="true" />
	<node name="director2" pkg="flircam" type="director2.py" />
</launch>
